web/modules/custom/routing/src/Controller/RoutingController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Drupal\routing\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Cache\Context\CacheContextInterface;
+
+/**
+ * Controller to display the username
+ */
+class RoutingController extends ControllerBase {
+
+  /**
+   * Class property for current user.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $account;
+
+  /**
+   * Constructs an account interface object
+   *
+   * @param AccountInterface $account
+   *   Holds details about the user account
+   */
+  public function __construct(AccountInterface $account) {
+    $this->account = $account;
+  }
+
+  /**
+   * This function will be used to retrieve data associated with account
+   * with key 'current_user'
+   *
+   * @param ContainerInterface $container
+   *   Instance of dependany injection container
+   *
+   * @return object
+   */
+  public static function create(ContainerInterface $container) {
+    //Instantiates the form class
+    return new static(
+      $container->get('current_user')
+    );
+  }
+  /**
+   * Prints the overview page
+   *
+   * @return array
+   *   Renderable array
+   */
+  public function myRoute() {
+    $userName = $this->account->getDisplayName();
+    $user = \Drupal\user\Entity\User::load(\Drupal::currentUser()->id());
```

Comment:
1/ Please use proper service to load an entity here.
2/ Please use dependency injection here.